### DISCUSSION TITLE
Speechly brower/OS compatibility

### DISCUSSION THREAD

**Discussion by jermlab** [speechly/speechly#280] [General]
I'm still getting my bearings with Speechly, so I apologize if I've overlooked something obvious.

I've found a few different versions of push-to-talk.js, and they all have issues with at least one major browser or OS from what I can tell. 

https://speechly.github.io/browser-ui/v2/push-to-talk-button.js
Windows firefox, OK
Windows Chrome, FAIL 
iOS,  OK

https://unpkg.com/@speechly/browser-ui@6.0.5/core/push-to-talk-button.js
Windows firefox, OK
Windows Chrome, OK 
iOS,  FAIL

I also tried https://speechly.github.io/browser-ui/latest/push-to-talk-button.js, but this also has problems. 

The strange thing is that if I include both of the following, it seems to work on everything. (A bit hacky though)
https://unpkg.com/@speechly/browser-ui@6.0.5/core/push-to-talk-button.js
https://speechly.github.io/browser-ui/v2/push-to-talk-button.js

It also seems that the documentation is no longer being supported. Does this mean that the push-to-talk.js will be depreciated at some point?

**Comment by bigdatabaracus** [comment]
Hello @jermlab, thanks for reporting and for providing the browser data 🙏  

Yes, we are sunsetting support for `push-to-talk.js` for the free tier. The recommendation is to build your own ui component logic on top either our [browser client](https://github.com/speechly/speechly/tree/main/libraries/browser-client) or [react client](https://github.com/speechly/speechly/tree/main/libraries/react-client) 

Like mentioned earlier [here](https://github.com/speechly/speechly/discussions/279#discussioncomment-4933372) the situation is different on the [payed tiers](https://www.speechly.com/pricing), where we can provide more comprehensive support also for the ui components. Please contact [hello@speechly.com](mailto:hello@speechly.com) if you'd like to hear more about those options.

**Comment by jermlab** [comment]
Thank you for your response. I will set that up and go from there.
